modules/candidate_profile/php/candidateinfo.class.inc

```diff
@@ -0,0 +1,38 @@
+<?php declare(strict_types=1);
+namespace LORIS\candidate_profile;
+
+/**
+ * The CandidateInfo class is used to represent a term to be
+ * displayed in the Candidate Info table of the candidate dashboard.
+ *
+ * It must be a simple term/value pair that can be encoded as a JSON
+ * object to be rendered to the client.
+ *
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ */
+class CandidateInfo implements \LORIS\GUI\Widget
+{
+    /**
+     * Construct a CandidateInfo object
+     *
+     * @param string $term  The term being summarized for the candidate
+     * @param string $value The value of the term
+     */
+    public function __construct(
+        string $term,
+        string $value
+    ) {
+        $this->term  = $term;
+        $this->value = $value;
```

Comment:
These aren't declared as class properties. I'm not sure if PHP will do this implicitly but they should probably be added and scoped at the top of the class.

Reply:
it's valid PHP but you're right, they should be declared anyways, eventually 'allow_missing_properties" should be set to false in phan..